### ISSUE TITLE
Exceptions not thrown on missing or read-only property set.

### ISSUE THREAD

**Issue by grokys** [cwensley/Portable.Xaml#66]
I created the following test program:

```
    class Program
    {
        static void Main(string[] args)
        {
            var xaml = "<Foo xmlns='clr-namespace:ConsoleApp2;assembly=ConsoleApp2' ReadOnly='hello' Missing='world'/>";

            var foo = XamlServices.Parse(xaml);
        }
    }

    public class Foo
    {
        public string ReadOnly { get; } = "Bar";
    }
```

There are 2 things wrong with the XAML here:

- Trying to set readonly property `Foo.ReadOnly`
- Trying to set non-existent property `Missing`

However program runs and the XAML loads without exception. If I run it with System.Xaml exceptions are thrown.

**Comment by cwensley** [comment]
Thanks for the issue report!  I believe this is fixed in #54.

I've added specific tests for these two exact cases though.

**Comment by grokys** [comment]
Thanks! Looks like that PR was merged after the last release, which would explain why I was still seeing it in 0.16.0. Closing.
